openSUSE-Tomcat-base-image/pom.xml

```diff
@@ -0,0 +1,146 @@
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.cafapi.opensuse.tomcat</groupId>
+        <artifactId>opensuse-tomcat-image-aggregator</artifactId>
+        <version>5.7.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>opensuse-tomcat-base-image</artifactId>
+    <packaging>pom</packaging>
+    
+    <name>openSUSE Tomcat base image</name>
+    <properties>
+        <maven.install.skip>true</maven.install.skip>
+        <DOCKER_HUB_PUBLIC>${dockerHubPublic}</DOCKER_HUB_PUBLIC>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>0.34.0</version>
+                <executions>
+                    <execution>
+                        <id>docker-build</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>start-docker-image</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>start</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>stop-docker-image</id>
+                        <phase>post-integration-test</phase>
+                        <goals>
+                            <goal>stop</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>docker-push</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                        <configuration>
+                            <filter>opensuse-tomcat-base-image</filter>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
+                    <images>
+                        <image>
+                            <alias>opensuse-tomcat-base-image</alias>
+                            <name>${dockerCafImagePrefix}opensuse-tomcat-base-image${dockerProjectVersion}</name>
```

Comment:
I don't think this is a good name for the final docker image.

Reply:
I'm not sure of another name to separate it from the image with tomcat - I dont think we should be changing that image name as I dont think should be affect downstream dependencies. If we do want to update the image with logback, then we'd need to update all downstreams

Reply:
Correct, the original image name should not be changed but I think a better one for this is needed as calling it `opensuse-tomcat-**base-image**` I think confuses it a bit with the original one as it is also a base image.

Reply:
Ok I didn't realise that the original image served as a base image for other projects